docs/API/panel/timeSeries/index.md

```diff
@@ -495,7 +495,7 @@ fieldConfig.defaults.custom.withSpanNulls(value)
 
 PARAMETERS:
 
-* **value** (`string`)
+* **value** (`boolean`,`number`)
 
```

Comment:
This again seems like a meaningful change, not just a docs re-render.

Reply:
same as above